README.md

```diff
@@ -4,8 +4,8 @@ Please see the original thread on [BoardGameGeek](https://boardgamegeek.com/thre
 
 This repository hosts **three** documents. Click to download the most recent builds:
 
-- [📜 **Rewritten Rule Book**](https://github.com/Heegu-sama/Homm3BG/raw/build/main.pdf/PDF/main.pdf)
-- [🖨️ **Rewritten Rule Book - printable version️️**](https://github.com/Heegu-sama/Homm3BG/raw/build/printable.pdf/PRINTABLE/printable.pdf)
+- [📜 **Rewritten Rule Book**](https://github.com/Heegu-sama/Homm3BG/raw/build/main_en.pdf/PDF/main_en.pdf)
+- [🖨️ **Rewritten Rule Book - printable version️️**](https://github.com/Heegu-sama/Homm3BG/raw/build/printable.p_en.pdf/PRINTABLE/printable_en.pdf)
 - [📋 **Comprehensive Components List**](https://github.com/Heegu-sama/Homm3BG/raw/build/components_list.pdf/COMPONENTS_LIST/components_list.pdf)
```

Comment:
do we want to include links to other translations' builds here as well?

Reply:
Yes we do, but we don't want to store them in the repo. With two languages, it will be over 300 MB of binary storage to pull after merging every PR. We don't want that. We'll have to find a way to store it externally.